app/assets/v2/js/kudos-search.js

```diff
@@ -32,7 +32,7 @@ function kudosSearch(elem) {
 
           let query = {
             term: params.term[0] === '@' ? params.term.slice(1) : params.term,
-            network: 'mainnet',
+            network: document.web3network,
```

Comment:
note that this will yieid complaints from end users who are not prompted to change their web3 netowrk when searching...

Reply:
@vporton do you want to address Kevin's comment here?

Reply:
Just revert my change.

Reply:
I have reverted the change.